autosubmit/platforms/pbsplatform.py

```diff
@@ -78,9 +69,6 @@ def __init__(self, expid, name, config, version):
     def create_a_new_copy(self):
         return PBSPlatform(self.expid, self.name, self.config, self.config[self.name.upper()].get('VERSION', ''))
 
-    def parse_queue_reason(self, output, job_id):
-        pass
```

Comment:
It's implemented in the parent. Not sure why they had a pass here... but probably safer to leave it as it was (PBS is broken anyway, this whole file can be reverted)